contrib/ansible/vagrant/README.md

```diff
@@ -0,0 +1,49 @@
+## Vagrant deployer for Kubernetes Ansible
+
+This deployer sets up a vagrant cluster and installs kubernetes with flannel on it.
+
+## Before you start !
+
+You will need to pick your vagrant provider. Currently supported are openstack, libvirt, and virtualbox.
```

Comment:
Can we add how to pick. Once I have picked do I have to take any actions? How do I configure that provider?